core/templates/domain/topic/editable-topic-backend-api.service.spec.ts

```diff
@@ -332,8 +331,7 @@ describe('Editable topic backend API service', () => {
 
     flushMicrotasks();
 
-    expect(successHandler).toHaveBeenCalledWith(
-      sampleDataResults.canonical_story_summary_dicts);
+    expect(successHandler).toHaveBeenCalled();
```

Comment:
Why is this changed?

Reply:
The value used previously `sampleDataResults.canonical_story_summary_dicts` had a value of null as `canonical_story_summary_dicts` has parent object `topic_dict`. I thought it would be better this way. Please help me if there is a correct apprach to be followed here.

Reply:
Can you just modify the `sampleDataResults.canonical_story_summary_dicts` or create a new variable for that and then test it with that?

Reply:
Done!